### PR TITLE
docs: add note on how to rotate client secret for windows code signing

### PR DESCRIPTION
**Pull request by jamilbk**
Moved the code signing to our US-based Azure account (firezoneprod.onmicrosoft.com)

### CODE REVIEW

File: rust/gui-client/README.md [hunk 1]
```diff
@@ -49,6 +49,27 @@ stat ../target/release/Firezone.exe
...
+secrets. This was originally set up using these guides for inspiration:
+
+- https://melatonin.dev/blog/how-to-code-sign-windows-installers-with-an-ev-cert-on-github-actions/
+- https://support.globalsign.com/code-signing/code-signing-using-azure-key-vault
+
+Renewing / issuing a new code signing certificate and associated Azure entities is outside the scope of this section. Use the guides above if this needs to be done.
+
+Instead, you'll most likely simply need to rotate the Azure `CodeSigning` Application's client secret.
+
+To do so, login to the Azure portal using your `@firezoneprod.onmicrosoft.com` account.
```
Comment (ReactorScram) [inline rust/gui-client/README.md]:
```suggestion
To do so, login to [the Azure portal](https://portal.azure.com) using your `@firezoneprod.onmicrosoft.com` account.
```

File: rust/gui-client/README.md [hunk 2]
```diff
@@ -49,6 +49,27 @@ stat ../target/release/Firezone.exe
...
+
+Then,
+
+- Find and navigate to the `CodeSigning` app registration.
```
Comment (ReactorScram) [inline rust/gui-client/README.md]: I cannot find this. I tried "Trusted Signing Accounts"

image


Almost everything I click in the Azure Portal, I get this thing about free trials. Am I in the right place?
image

**Review by ReactorScram — COMMENTED** [review]
Needs URLs, I'm totally lost on trying to even find where this stuff is

File: rust/gui-client/README.md (hunk 2 above)
Comment (jamilbk) [reply rust/gui-client/README.md]: Huh, that's not right. Let me double check. Thanks for actually trying these steps!

Comment (jamilbk) [reply rust/gui-client/README.md]: Can you give it a shot now? I think I need to add you to the necessary Roles. Still figuring out what those are.

Comment (ReactorScram) [reply rust/gui-client/README.md]: Now I can get into `Trusted Signing Accounts` (Whose URL includes `Microsoft.CodeSigning` so I'm not sure what it's really called lol), but I can't see any.
image

Nothing in `App Services` either
image

Comment (jamilbk) [reply rust/gui-client/README.md]: Ah, it's in the Entra ID service. Go to Entra ID and then App Registrations on the left.

**Comment by jamilbk** [comment]
Clarified where to find the App registration.

File: rust/gui-client/README.md [hunk 3]
```diff
@@ -49,6 +49,28 @@ stat ../target/release/Firezone.exe
...
+
+Renewing / issuing a new code signing certificate and associated Azure entities is outside the scope of this section. Use the guides above if this needs to be done.
+
+Instead, you'll most likely simply need to rotate the Azure `CodeSigning` Application's client secret.
+
+To do so, login to [the Azure portal](https://portal.azure.com) using your `@firezoneprod.onmicrosoft.com` account.
+
+Then,
+
+- Go to the `Entra ID` service
```
Comment (ReactorScram) [inline rust/gui-client/README.md]:
```suggestion
- Go to the `Microsoft Entra ID` service
```

**Review by ReactorScram — APPROVED** [review]

File: rust/gui-client/README.md [hunk 4]
```diff
@@ -49,6 +49,28 @@ stat ../target/release/Firezone.exe
...
+- https://melatonin.dev/blog/how-to-code-sign-windows-installers-with-an-ev-cert-on-github-actions/
+- https://support.globalsign.com/code-signing/code-signing-using-azure-key-vault
+
+Renewing / issuing a new code signing certificate and associated Azure entities is outside the scope of this section. Use the guides above if this needs to be done.
+
+Instead, you'll most likely simply need to rotate the Azure `CodeSigning` Application's client secret.
+
+To do so, login to [the Azure portal](https://portal.azure.com) using your `@firezoneprod.onmicrosoft.com` account.
+
+Then,
```
Comment (thomaseizinger) [inline rust/gui-client/README.md]:
```suggestion
To do so, login to [the Azure portal](https://portal.azure.com) using your `@firezoneprod.onmicrosoft.com` account.
Try to access it via the following [deep-link](https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/~/Credentials/appId/51da0daa-39dd-4890-9018-e02609efc9c8).
If that doesn't work:


```

**Review by thomaseizinger — APPROVED** [review]

File: rust/gui-client/README.md [hunk 5]
```diff
@@ -49,6 +49,28 @@ stat ../target/release/Firezone.exe
...
+
+- Go to the `Microsoft Entra ID` service
+- Find and navigate to the `CodeSigning` app registration
```
Comment (bmanifold) [inline rust/gui-client/README.md:73]:
```suggestion
- Click on `App Registrations`
- Make sure the tab `All applications` is selected
- Find and navigate to the `CodeSigning` app registration
```

**Review by bmanifold — APPROVED** [review]

**Pull request merged**